axe.d.ts

```diff
@@ -155,6 +155,13 @@ declare namespace axe {
 		}[];
 		cleanup?(callback: Function): void;
 	}
+	interface GetRuleResult {
```

Comment:
I think `RuleMetadata` would be appropriate as that's what we call the JSON files that create this data.